SUPPORT.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is it left blank intentionally ?

Reply:
Added it. I think I missed it out while copying.